core/felt/felt.go

```diff
@@ -26,7 +26,6 @@ const (
 )
 
 const (
-	Limbs = fp.Limbs // number of 64 bits words needed to represent a Element
```

Comment:
I am not sure why this is being removed.

Reply:
it was used only in migration I thought that it was introduced only for migrations